src/main/java/com/contrastsecurity/maven/plugin/InstallAgentContrastMavenMojo.java

```diff
@@ -53,11 +63,20 @@ public String computeAppVersion(Date currentDate) {
             getLog().info("No CI build number detected, we'll use current timestamp.");
             appVersionQualifier = new SimpleDateFormat("yyyyMMddHHmmss").format(currentDate);
         }
-        computedAppVersion = appName + "-" + appVersionQualifier;
+        if (StringUtils.isNotBlank(appId)) {
```

Comment:
See comments above. Let's not modify this behavior. Get the app from the SDK, then build the app version using the appName.

Reply:
Done.